Subject: [PATCH] build adjustments
removing Qt::GuiPrivate for a try
---
Index: gui/CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/gui/CMakeLists.txt b/gui/CMakeLists.txt
--- a/gui/CMakeLists.txt	(revision 8dd2381bb5b8f4c91dfcea0395298e47d32bf063)
+++ b/gui/CMakeLists.txt	(revision e733ab2925a5f978dd33d1cc79bdc20146a6952b)
@@ -90,6 +90,16 @@
 )
 target_include_directories(chiaki PRIVATE include)
 
+target_include_directories(chiaki PRIVATE
+		include
+		# C:\msys64\mingw64\include\qt6\QtGui\6.10.0\QtGui\qpa
+		"C:/msys64/mingw64/include/qt6/QtGui/${Qt6Gui_VERSION}/QtGui"
+		"C:/msys64/mingw64/include/qt6/QtGui/${Qt6Gui_VERSION}/QtGui/private"
+		"C:/msys64/mingw64/include/qt6/QtGui/${Qt6Gui_VERSION}/QtGui/qpa"
+)
+
+message(STATUS "C:/msys64/mingw64/include/qt6/QtGui/${Qt6Gui_VERSION}/QtGui")
+
 add_library(placebo-libav-impl src/libav_impl.c)
 target_link_libraries(placebo-libav-impl PkgConfig::LIBPLACEBO FFMPEG::avcodec FFMPEG::avutil FFMPEG::avformat)
 target_link_libraries(chiaki PkgConfig::LIBPLACEBO placebo-libav-impl)
@@ -100,7 +110,7 @@
 	target_link_libraries(chiaki chiaki-cli-lib)
 endif()
 
-target_link_libraries(chiaki Qt::Core Qt::Gui Qt::GuiPrivate Qt::Concurrent Qt::Svg Qt::Qml Qt::Quick Qt::Widgets)
+target_link_libraries(chiaki Qt::Core Qt::Gui Qt::Concurrent Qt::Svg Qt::Qml Qt::Quick Qt::Widgets)
 if (${Qt6DBus_FOUND} AND NOT WIN32)
 	target_link_libraries(chiaki Qt::DBus)
 if (NOT APPLE)
Index: lib/CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lib/CMakeLists.txt b/lib/CMakeLists.txt
--- a/lib/CMakeLists.txt	(revision 9c5aacb8f17648d9982c36db4a4d2127d459c4f7)
+++ b/lib/CMakeLists.txt	(revision e733ab2925a5f978dd33d1cc79bdc20146a6952b)
@@ -188,4 +188,6 @@
 if(CHIAKI_ENABLE_TESTS)
 	add_executable(holepunch-test include/chiaki/remote/holepunch.h src/remote/holepunch-test.c)
 	target_link_libraries(holepunch-test chiaki-lib)
-endif()
\ No newline at end of file
+endif()
+
+target_link_libraries(chiaki-lib wldap32)
Index: scripts/build-libplacebo-windows.sh
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/scripts/build-libplacebo-windows.sh b/scripts/build-libplacebo-windows.sh
--- a/scripts/build-libplacebo-windows.sh	(revision 9c5aacb8f17648d9982c36db4a4d2127d459c4f7)
+++ b/scripts/build-libplacebo-windows.sh	(revision e733ab2925a5f978dd33d1cc79bdc20146a6952b)
@@ -13,6 +13,6 @@
 cd libplacebo || exit 1
 git checkout $TAG || exit 1
 DIR=./build || exit 1
-meson setup --prefix /mingw64 -Dxxhash=disabled $DIR || exit 1
+meson setup --wipe --prefix /mingw64 -Dxxhash=disabled $DIR || exit 1
 ninja -C$DIR || exit 1
 ninja -Cbuild install || exit 1
